build.gradle

```diff
@@ -165,3 +213,50 @@ task createClasspathManifest {
 dependencies {
     testRuntime files(createClasspathManifest)
 }
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_KEY')
+    pkg {
+        repo = "maven"
+        name = 'scala-multiversion-plugin'
+        userOrg = 'adtran'
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/ADTRAN/gradle-scala-multiversion-plugin.git'
+        websiteUrl = 'https://github.com/ADTRAN/gradle-scala-multiversion-plugin.git'
+        issueTrackerUrl = 'https://github.com/ADTRAN/gradle-scala-multiversion-plugin/issues'
+        labels = ['gradle', 'scala', 'versioning']
+        publications = ['mavenPublication']
+        publish = false
+        version {
+          name = "${project.version}"
+          vcsTag = "${project.version}"
+          released = new Date()
+          attributes = [
+            'gradle-plugin': 'com.adtran.betablocker-plugin:com.adtran:scala-multiversion-plugin'
+          ]
+          gpg {
+              sign = System.env['GPG_PASS'] != null ? true : false
```

Comment:
The ternary seems pointless here.

Reply:
:+1: Was mostly for local testing. I can just set it to true now.